### ISSUE TITLE
List size is not announced with Sayall command. 

### ISSUE THREAD

**Issue by 7dITS** [FreedomScientific/VFO-standards-support#366]
### Summary

> Brief description of the issue. Please list any specific steps.

The size of the list is not being announced by jaws. 
This occurs when the sayall command is issued. 

A form that I am working on displays errors in a list at the top of the page in a div. I provided a sample image below. Jaws is announcing the heading and then announces the list items. It says "list 1. the 9-digit ID number is invalid, 2. a month has not been selected" Jquery is used to display these errors. 
> 
> Example:
![image](https://user-images.githubusercontent.com/24323027/73959039-c72c6c80-48d6-11ea-80d4-825f0de03b52.png)

### Expected result

> What did you expect to happen?
I expect jaws to announce the size of the list when the sayall command is issued. For example "ordered list of 4 items" before it reads the first item of the list. 

### Actual result

> What actually did happen?
When the say all command is issued the size of a list of items is not announced. 

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

N/A

#### JAWS version and build number
 #2019.1912.1.400

 
#### Operating System and version
Windows 10 Enterprise  1803

#### Browser and version:
Firefox ESR 68.4.2
IE 11 11.1246.17134.0

**Comment by JeremiahRogersVispero** [comment]
@7dITS I cannot reproduce this admittedly old issue. If you can still reproduce it, please share a page where the issue exists.
